### PR TITLE
Estonian translation

### PR DESCRIPTION
**Pull request by Repsionu**
Another try :-)
Best, JR

### CODE REVIEW

File: src/translations/et.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "avalanches": "laviin",
+    "rain": "sademed",
+    "flood": "üleujutus",
+    "rain_flood": "paduvihm"
+  },
+  "messages": {
+    "yellow": "Kollane hoiatus: {0}",
+    "orange": "Oranž hoiatus: {0}",
+    "red": "Punane hoiatus: {0}"
+  }
```
Comment (MrBartusek) [inline src/translations/et.json:29]: Remove double dot `:` or replace with the equivalent `for` to maintain continuity of the sentence

**Review by MrBartusek — CHANGES_REQUESTED** [review]
Hey, thanks for contributing! Please update your branch with new commits, resolve merge conflict if there will be any, and fix that one thing with translations. Then this PR will be merged.

**Comment by Repsionu** [comment]
Can there be other delimiters like '-' in translation for red/yellow etc?
Best, JR

File: src/translations/et.json (hunk 1 above)
Comment (Repsionu) [reply src/translations/et.json]: Done

**Comment by MrBartusek** [comment]
> Can there be other delimiters like '-' in translation for red/yellow etc?
> Best, JR

The idea behind these messages is to make them feel like real sentences. It will be best to keep it as is.

File: src/localize.js [hunk 2]
```diff
@@ -4,9 +4,12 @@
...
 import * as fr from './translations/fr.json';
 import * as pl from './translations/pl.json';
+import * as et from './translations/et.json';
 
 var languages = {
 	en,
+	pl,
+	et,
 	fr,
 	pl
```
Comment (MrBartusek) [inline src/localize.js:14]: `pl` is specified two times

**Review by MrBartusek — CHANGES_REQUESTED** [review]

**Review by MrBartusek — APPROVED** [review]

**Pull request merged**